### ISSUE TITLE
Pivot Total column is not displayed when only one pivot column

### ISSUE THREAD

**Issue by vitolpoc** [infinite-table/infinite-react#22]
When configuring only one pivot and the `pivotTotalColumnPosition`, the total column is not displayed.

sandbox: https://codesandbox.io/s/infinite-table-pivot-total-column-for-one-level-m3ioz?file=/src/App.tsx

Version: 0.0.8
Actual Result: Total column is not displayed
Expected Result: Total column should be displayed if configured with "start" or "end" value

**Comment by roblotter** [comment]
This was intentionally implemented like this, as there are no column groups and only one pivot available.
<img width="658" alt="Screenshot 2022-01-03 at 12 13 14" src="https://user-images.githubusercontent.com/50020244/147919373-3f3749c5-e6ed-483a-95c3-f4dd78092f06.png">

We can revisit this if you think it would be useful to allow a totals column in this case - our initial thinking was that you don't have column groups generated, so no need for a totals column to aggregate the column group.

**Comment by vitolpoc** [comment]
In the screenshot above, we cannot have the aggregation (let's say the average) for the whole TypeScript row then. 

The fact that the column represents a pivot means it's a group of values, so it makes sense to be able to configure the whole aggregation for this group like if it was in another column group IMO.

**Comment by roblotter** [comment]
So basically you want a grand total column, correct?

Something like
![Screenshot 2022-01-06 at 13 36 51](https://user-images.githubusercontent.com/50020244/148377188-2fdf832a-3447-4518-84f8-51d2b7b71cc7.png)
([source](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/RunningTotals/jQuery/Light/))

(ignore for now the "Grand Total" row - that will probably also come in the future)

What's nice would also be the ability to have multiple aggregations - I think this is what you mean by #19 , am I right?

Let us know if our assumptions are correct, so we can start on the implementation

**Comment by vitolpoc** [comment]
yes your assumptions are correct. Like we could have a total row, we could have a total column (when using pivot).

Indeed #19 refers to multiple aggregations.
